flight/Modules/Stabilization/inc/virtualflybar.h

```diff
@@ -43,4 +43,4 @@ int stabilization_virtual_flybar_pirocomp(float z_gyro, float dT);
 /**
  * @}
  * @}
- */
+ */
```

Comment:
> A source file that is not empty shall end in a new-line character,
which shall not be immediately preceded by a backslash character before any such
splicing takes place.

For anyone playing along at home :)